Library/.rubocop_shared.yml

```diff
@@ -0,0 +1,7 @@
+AllCops:
```

Comment:
What’s the point of this file? Given the cask file already duplicates much it seems strange to inherit to save the few lines this does when running ‘rubocop’ outside of ‘brew cask style’ in an editor will return wrong results again.

Reply:
> when running ‘rubocop’ outside of ‘brew cask style’ in an editor will return wrong results again.

It doesn't return wrong results, each tap has a config file inheriting from `.rubocop_cask.yml`.